### ISSUE TITLE
Product Time Synchronisation problem in WooCommerce and Facebook Catalog

### ISSUE THREAD

**Issue by Suzparis** [woocommerce/facebook-for-woocommerce#1755]
## 🔎 Isolate the bug

- [X] I have confirmed this occurs in the most recent version of WordPress, WooCommerce, and Facebook for WooCommerce.
- [ ] I have confirmed this occurs when only WooCommerce and Facebook for WooCommerce are active and when using a default WordPress or WooCommerce theme.

## ✍️ Describe the bug

##🚶‍♀️ Steps to reproduce

1.For a variable product, In WooCommerce, put a start date 21/01/2021 , you will find inside the Facebook catalogue the day before 20/01/2021

https://1drv.ms/u/s!Ak_IFdTmvN05gi4U6pDaZX2Z-Y0m?e=bRMPW2
https://1drv.ms/u/s!Ak_IFdTmvN05gjAAhuPc-zQc-AH9?e=iJdd8d

**Comment by jconroy** [comment]
Thanks @Suzparis 

Just tried to do something similar and it does appear there might be an issue - or at least some confusion.

I've labeled as a bug to investigate and/or understand what is happening

---

My test store is set to Adelaide time +10:30 UTC

I set some sale price dates as follows

![Screen Shot 2021-02-18 at 11 10 42 am](https://user-images.githubusercontent.com/355014/108286981-01169b00-71da-11eb-911c-d70ea3ca320d.png)

And see

![Screen Shot 2021-02-18 at 11 08 32 am](https://user-images.githubusercontent.com/355014/108286994-070c7c00-71da-11eb-8811-8bccc116135d.png)

Date format used is https://en.wikipedia.org/wiki/ISO_8601

At the very least with a start date of the 9th for me I'd expect to see a start date of the 8th (not the 7th) in a -0800 offset

**Comment by Suzparis** [comment]
Hi @jconroy,
Thanks for your time and your help. I tried and I checked everything, in order to be sure to have the same time zone between Facebook and WooCommerce and it is. I also updated wordpress to last version but nothing change.
I always have a problem of start date one day before.
WooCommerce - Start date : 22-02-2021
Facbook : 21-02-2021 00:00

![Capture d’écran 2021-02-18 à 12 45 27](https://user-images.githubusercontent.com/77985887/108352859-bdc32900-71e7-11eb-9d8a-9e3b03b37edd.png)
<img width="636" alt="Capture d’écran 2021-02-18 à 12 45 43" src="https://user-images.githubusercontent.com/77985887/108352887-c3b90a00-71e7-11eb-9e33-5751e692b88a.png">
